multiversx_sdk/core/constants.py

```diff
@@ -23,7 +20,7 @@
 TOKEN_RANDOM_SEQUENCE_LENGTH = 6
 
 DEFAULT_MESSAGE_VERSION = 1
-BECH32_ADDRESS_LENGTH = 62
+HEX_ADDRESS_LENGTH = 64
```

Comment:
Removed constant is a bit of breaking change, but should be fine.

Reply:
indeed, reverted.